src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -299,7 +299,7 @@ private ExpressionSyntax SkipParensAndNullSuppressions(ExpressionSyntax e)
             BoundExpression convertedExpression = ConvertPatternExpression(
                 inputType, patternExpression, expression, out constantValueOpt, hasErrors, diagnostics);
 
-            ConstantValueUtils.CheckLangVersionForConstantValue(expression, diagnostics);
+            ConstantValueUtils.CheckLangVersionForConstantValue(convertedExpression, diagnostics);
```

Comment:
This is an unrelated bug?

Reply:
> This is an unrelated bug? Or just expressing itself now?

Neither. Previously, `BoundInterpolatedString` was never converted, so `expression` and `convertedExpression` were the same node.